net/iperf3/files/etc/config/iperf3

```diff
@@ -0,0 +1,16 @@
+
+config server wan
+	option enabled '1'
+	option port '5202'
+	option iface 'wan'
+	option redirect '1'
+	option interval '1'
+
+#config client internet
```

Comment:
Do you think it would be a good idea to have a config for one of the public iperf3 servers in the disabled state rather than commented out config?

Reply:
Sounds good, I will look for a server on the internet and use this as client config.